### PR TITLE
feat(components): add more share services & redraw icons

### PR DESCRIPTION
**Pull request by CN-ycx**
I plan to add more services than now, but:
Some service is not providing public share URL, or need api id, or using url scheme like `messenger://`, it hard to find.

@Mister-Hope We can discuss how to add these services, and add more later.

### CODE REVIEW

File: docs/components/src/zh/guide/utilities/share.md [hunk 1]
```diff
@@ -84,14 +93,23 @@ description: 分享组件，将页面内容分享到社交媒体。
...
+    | "sms"
+    | "snapchat"
     | "telegram"
+    | "tumblr"
     | "twitter"
+    | "vk"
+    | "weibo"
     | "whatsapp"
-    | "weibo";
+    | "wordpress;
```
Comment (Mister-Hope) [inline docs/components/src/zh/guide/utilities/share.md:112]: typo here

File: docs/components/src/guide/utilities/share.md [hunk 2]
```diff
@@ -84,14 +93,23 @@ If you are using twitter, you can set `componentOptions.share.twitterUserName` w
...
+    | "sms"
+    | "snapchat"
     | "telegram"
+    | "tumblr"
     | "twitter"
+    | "vk"
+    | "weibo"
     | "whatsapp"
-    | "weibo";
+    | "wordpress;
```
Comment (Mister-Hope) [inline docs/components/src/guide/utilities/share.md:112]: typo here

**Review by Mister-Hope — CHANGES_REQUESTED** [review]
We should also update the default value,  Um for example people will rarely share with sms.

The default value will be a bunch of popular services

**Comment by CN-ycx** [comment]
Work complete, most of sharing services not modify the link, but also then not provide docs.
They mostly from search results or application build-in sharing.
If necessary, I will provide link of docs for sharing services with API key requirement.

**Comment by Mister-Hope** [comment]
Thanks, can you create another pr with https://vueuse.org/core/useShare/#useshare?

This could be a new root component called `<ShareButton>`, which it should be placed above the backtotop button.

**Comment by Mister-Hope** [comment]
> We should also update the default value, Um for example people will rarely share with sms.
>
> The default value will be a bunch of popular services

image

I did not see the above changes? My suggestions is to update here, it's not reasonable to have a full list of services by default which contains very uncommon ones.

You can try to look around for popular websites to see their share services and leave 5-8 popular service here.

Note: JSDoc of the option should be updated as well besides docs about the default fallback value.

**Comment by Mister-Hope** [comment]
And again, thanks for your contirbution😉

**Comment by CN-ycx** [comment]
It's my fault, didn't understand so I only modified the order, sorry😰

**Pull request merged**